src/app/notifications/notifications.page.html

```diff
@@ -1,14 +1,23 @@
-<ion-header [translucent]="true">
-  <ion-toolbar>
-    <ion-title>Notifications</ion-title>
-  </ion-toolbar>
-</ion-header>
-
 <ion-content [fullscreen]="true">
-  <ion-header collapse="condense">
-    <ion-toolbar>
-      <ion-title size="large">Notifications</ion-title>
-    </ion-toolbar>
-  </ion-header>
-  <app-explore-container name="Notifications"></app-explore-container>
-</ion-content>
+  <div class="page-title">
+    <app-header
+      [headerTitle]="headerTitle"
+      [backButton]="backButton"
+      [defaultHref]="defaultHref"
+    ></app-header>
+    <app-search
```

Comment:
Do we have also the filters?

Reply:
There will be, just need to make a function on the backend, because now I checked that there is not for the moment